src/metrics/mod.rs

```diff
@@ -91,10 +91,16 @@ impl RecentlyAccessedReleases {
     }
 
     pub(crate) fn record(&self, krate: i32, version: i32, target: &str) {
-        self.crates.insert(krate, Instant::now());
-        self.versions.insert(version, Instant::now());
+        if self.platforms.len() > 100_000 {
+            // Avoid filling the maps _too_ much, we should never get anywhere near this limit
```

Comment:
```suggestion
            // Avoid filling the maps _too_ much, we should never get anywhere near this limit
            debug!("more than 100_000 hits in the last hour, disabling further metrics");
```

Reply:
We will see it capping out on the metrics, I think it would be a waste to spam a message per-request to the logs too.